### ISSUE TITLE
"Updating Steam" window doesn't have a WM_CLASS set

### ISSUE THREAD

**Issue by tadzik** [ValveSoftware/steam-for-linux#3658]
http://i.imgur.com/IdTs8Nc.png

This prevents window managers from automatically assigning it to a specific workspace, which otherwise works with the rest of the steam client.

**Comment by Plaque-fcc** [comment]
Weird, for AwesomeWM it works fine:

{ rule = { class = "Steam" },
      properties = { tag = tags[1][8]  } },

Don't know why.

**Comment by tadzik** [comment]
Hmm, I'm using i3 and that particular updating window did not get automatically sent to 10 for some reason. Perhaps awesome falls back to WM_NAME when class is not present.

**Comment by Plaque-fcc** [comment]
Hah, I misread the title.

Yeah, can confirm this, other windows do have such property set, while
Updater window doesn't.

**Comment by Plaque-fcc** [comment]
Yes, it has WM_NAME(UTF8_STRING) = "Steam" so this can be a workaround
for a while.

**Comment by Suseika** [comment]
Have the same issue, this is what `xprop` says about that window:
```
$ xprop
_NET_WM_DESKTOP(CARDINAL) = 1
WM_STATE(WM_STATE):
                window state: Normal
                icon window: 0x0
WM_NORMAL_HINTS(WM_SIZE_HINTS):
                user specified location: 640, 460
                user specified size: 400 by 129
                program specified minimum size: 400 by 129
                program specified maximum size: 400 by 129
                window gravity: Center
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW
WM_NAME(UTF8_STRING) = "Steam"
STEAM_BIGPICTURE(CARDINAL) = 1
```
No `WM_CLASS`, sadly.

**Comment by StaticRocket** [comment]
Bumping this because I am also trying to use the WM_CLASS field for window manager rules and am currently being foiled by the update window. This issue is still prevalent.

**Comment by awsms** [comment]
The absolute state.
